src/Utils/utils.ts

```diff
@@ -57,17 +57,17 @@ export const formatName = (
   },
   hidePrefixSuffix: boolean = false,
 ) => {
-  return (
-    [
-      hidePrefixSuffix ? undefined : user.prefix,
-      user.first_name,
-      user.last_name,
-      hidePrefixSuffix ? undefined : user.suffix,
-    ]
-      .map((s) => s?.trim())
-      .filter(Boolean)
-      .join(" ") || user.username
-  );
+  if (!user) return "-";
+  const name = [
+    hidePrefixSuffix ? undefined : user.prefix,
+    user.first_name,
+    user.last_name,
+    hidePrefixSuffix ? undefined : user.suffix,
+  ]
+    .map((s) => s?.trim())
+    .filter(Boolean)
+    .join(" ");
+  return name || user.username || "-";
```

Comment:
not a big deal, but user would always have a username anyhow, so the `-` fallback would never get triggered in this case.

```suggestion
  return name || user.username;
```